src/views/CoCurricularTranscript/Components/Experience/index.tsx

```diff
@@ -35,4 +42,11 @@ const formatDuration = ({ Job_Start_Date, Job_End_Date }: StudentEmployment) =>
   }
 };
 
+const newJobTitle = ({ Job_Department_Name, Job_Title }: StudentEmployment) => {
+  if (Job_Title === '') {
+    return '';
+  }
+  return Job_Title === '' ? Job_Title : `${Job_Department_Name}, ${Job_Title}`;
```

Comment:
```suggestion
  if (Job_Title === '') {
    return '';
  }
  return Job_Title === '' ? Job_Title : `${Job_Department_Name}, ${Job_Title}`;
```
that top part and the first half of the ternary are logically identical. Is there a reason both are included?